tools/ci_build/build.py

```diff
@@ -1516,11 +1516,17 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
                     run_subprocess([sys.executable, '-m', 'unittest', 'discover', '-s', 'quantization'],
                                    cwd=cwd, dll_path=dll_path)
                     if args.enable_transformers_tool_test:
+                        import numpy
+                        numpy_init_version = numpy.__version__
                         required = {
-                            'numpy==1.19.2', 'coloredlogs==15.0', 'tf2onnx==1.8.5', 'transformers==4.6.1',
-                            'torch==1.8.1', 'tensorflow==2.5.0', 'onnxconverter-common==1.8.1', 'psutil'}
+                            'numpy==1.19.2', 'coloredlogs==15.0', 'transformers==4.6.1', 'torch==1.8.1',
```

Comment:
Can torch be removed from the list?

Reply:
torch is used in test_gelu_fusions.py:
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/python/transformers/test_gelu_fusions.py#L10

Reply:
Could you please put these things in a separated file named "requirements.txt", then let pip use that file directly? So that component governor  can track the dependencies better.